ENIGMAsystem/SHELL/Universal_System/Resources/fontinit.cpp

```diff
@@ -67,8 +67,8 @@ void exe_loadfonts(FILE* exe) {
 
     const unsigned int size = twid * thgt;
 
-    unsigned char* pixels =
-        new unsigned char[size * 4 + 1];  //FYI: This variable was once called "cpixels." When you do compress them, change it back.
+    int* pixels =
+        new int[size + 1];  //FYI: This variable was once called "cpixels." When you do compress them, change it back.
```

Comment:
@fundies you had broken fonts here, this fixes them. I would just leave it as int because it simplifies the for loop below it, there is nothing harmful in the later cast. Otherwise you'd have to change the 1 line in the for loop to 4 lines to assign each channel, I think it's nice the way Josh had it.